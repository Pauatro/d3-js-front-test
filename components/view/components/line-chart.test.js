import createViewContainer from './view-container.js'
import createLineChart from './line-chart.js'


it('renders correctly', () => {

    const container = document.createElement('div')
    container.style = "width: 300px; height: 200px;"

    document.body.append(container)

    const {
        svg,
        containerHeight
    } = createViewContainer(container, 0)

    const outerRadius = containerHeight/3.5
    const innerRadius = outerRadius*0.9
    const timeLine = [100, 200, 300, 400, 500, 600]
    createLineChart(svg, innerRadius, timeLine,0)
    expect(svg).toMatchSnapshot();
});