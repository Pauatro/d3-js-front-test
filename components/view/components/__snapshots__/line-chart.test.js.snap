// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Selection {
  "_groups": Array [
    Array [
      <svg
        viewBox="0,0,0,0"
      >
        <g
          transform="translate(0,2)"
        >
          <path
            d="M0,0L0,0Z"
            style="
            stroke: #93d55d;
            stroke-width: 1px;
            fill: #93d55d;
            fill-opacity: 0.2;
        "
          />
        </g>
      </svg>,
    ],
  ],
  "_parents": Array [
    null,
  ],
}
`;
